aleph-client/src/elections.rs

```diff
@@ -0,0 +1,74 @@
+use codec::Decode;
+use primitives::CommitteeSeats;
+use sp_core::H256;
+use substrate_api_client::AccountId;
+
+use crate::{AnyConnection, SignedConnection};
+
+pub fn get_committee_seats<C: AnyConnection>(
+    connection: &C,
+    block_hash: Option<H256>,
+) -> CommitteeSeats {
+    connection
+        .as_connection()
+        .get_storage_value("Elections", "CommitteeSize", block_hash)
+        .expect("Failed to decode CommitteeSize extrinsic!")
+        .unwrap_or_else(|| {
+            panic!(
+                "Failed to obtain CommitteeSize for block hash: {:?}.",
+                block_hash
+            )
+        })
+}
+
+pub fn get_validator_block_count<C: AnyConnection>(
+    connection: &C,
+    account_id: &AccountId,
+    block_hash: Option<H256>,
+) -> Option<u32> {
+    connection
+        .as_connection()
+        .get_storage_map(
+            "Elections",
+            "SessionValidatorBlockCount",
+            account_id,
+            block_hash,
+        )
+        .expect("Failed to decode SessionValidatorBlockCount extrinsic!")
+}
+
+#[derive(Decode)]
+pub struct EraValidators {
+    pub reserved: Vec<AccountId>,
+    pub non_reserved: Vec<AccountId>,
+}
+
+pub fn get_current_era_validators(connection: &SignedConnection) -> Vec<AccountId> {
```

Comment:
You are not using it anywhere.

Reply:
Fixed as part of the reorganization of the logic between `get_current_era_validators`, `get_current_era_reserved_validators` and `get_current_era_non_reserved_validators`.